python/lsst/validate/base/output.py

```diff
@@ -0,0 +1,18 @@
+# See COPYRIGHT file at the top of the source tree.
+from __future__ import absolute_import, division, print_function
+
+import json
+
+from .measurement import MeasurementSet, Measurement
+
+
+def output_measurements(package_name, measurement_dict):
```

Comment:
I'm losing my mind, sorry. You're right that `output_measurements` as a shim to the Butler is probably a good thing. But keeper in mind that there's more to a measurement than the value (need to connect to blobs, and to Job data).

Previously I used `Job.json` to recursively serialize all Measurement, Blob, and Job data via their `json` attributes. So I think we'll want to think about how to bring that idea back.

(I understand that right now MeasurementSet is effectively being serialized via it's `__str__` method rather than a `json` attribute calling `Measurement` `json` attributes.)

I'm fine if we merge this now and then re-group with more concrete design thinking on persistence.